### PR TITLE
Add Wechat provider

### PR DESCRIPTION
**Pull request by Darmody**
Add Wechat provider. 😄 

Resolved #27

### CODE REVIEW

File: lib/generators/sorcery/templates/initializer.rb [hunk 1]
```diff
@@ -121,6 +121,10 @@
   # config.paypal.callback_url = "http://0.0.0.0:3000/oauth/callback?provider=paypal"
   # config.paypal.user_info_mapping = {:email => "email"}
   #
+  # config.wechat.key = "
```
Comment (dankimio) [inline lib/generators/sorcery/templates/initializer.rb]: Can you add the closing quotation marks? So that it works after uncommenting.

Comment (Darmody) [inline lib/generators/sorcery/templates/initializer.rb]: Sure, updated.

**Comment by Ch4s3** [comment]
This seems fine if @dankimio and @athix are ok with it. I know we're talking about moving providers to separate gems, but we haven't moved forward on that, and wechat is very popular so a lot of folks might find it useful.

**Comment by Darmody** [comment]
@athix I have test it with my own app, it works!

And CHANGELOG.md updated.

**Pull request merged**